src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -399,6 +399,13 @@ void NotationViewInputController::mouseMoveEvent(QMouseEvent* event)
 
         m_view->notationInteraction()->drag(m_interactData.beginPoint, logicPos, mode);
         return;
+    } else if (m_interactData.hitElement == nullptr && (keyState & (Qt::ShiftModifier | Qt::ControlModifier))) {
+        if (!m_view->notationInteraction()->isDragStarted()) {
+            m_view->notationInteraction()->startDrag(std::vector<Element*>(), QPoint(), [](const Element*) { return false; });
+        }
+        m_view->notationInteraction()->drag(m_interactData.beginPoint, logicPos, 
+            (keyState & Qt::ControlModifier) ? DragMode::LassoList : DragMode::BothXY);
```

Comment:
NB this means that holding both shift and control down results in a list selection too.

Reply:
(and is a minor code style issue, see the check that now ran)

Reply:
> (and is a minor code style issue, see the check that now ran)

Seems uncrustify can't handle that syntax properly unless I remove the parenthesis.